### ISSUE TITLE
오류 명세 관련 문의드립니다.

### ISSUE THREAD

**Issue by ByulKim** [navertalk/chatbot-api#170]
안녕하세요.
api 호출 후 응답으로 아래와 같은 에러 메세지를 전달 받았습니다.

```
{
  "success": false,
  "resultCode": "01",
  "resultMessage": "현재 이 사용자에게 메시지를 보낼 수 없음"
}
```

이와 같은 경우, 오류 명세에서 [Authorization 정보가 잘못되었거나 이미 만료된 정보를 사용한 경우] 로 확인하였는데
관련하여 해결 방법이나, 자세한 오류를 확인할 수 있는 방법이 있을지 문의드립니다.

감사합니다.

**Comment by navertalk** [comment]
안녕하세요.
네이버 톡톡입니다.

프로필 API 문서에 아래와 같은 내용이 있는데요. ([링크](https://github.com/navertalk/chatbot-api/blob/master/profile_api_v1.md#profile-api-%EB%8F%99%EC%9E%91-%ED%9D%90%EB%A6%84))
​
<img width="1021" alt="Screen Shot 2021-12-01 at 2 27 25 PM" src="https://user-images.githubusercontent.com/30263758/144178239-a1c7341c-80e5-445a-b8d0-e2b61ac7d85e.png">

프로필 API를 호출하신 뒤, 사용자가 [동의]하거나 [취소]할 때까지 최대 3분가량의 대기 시간이 있어야 합니다.
최초에 1분으로 지정했으나 약관동의 절차 흐름이 변경된 것과 사용자가 약관을 읽어보거나 고민할 시간이 필요하다는 판단으로 시간이 늘어났습니다.
이 시간동안은 챗봇이 보내는 이벤트를 전달하지 않고 오류로 응답하고 있습니다.
​
감사합니다.

**Comment by ByulKim** [comment]
네, 사용자가 동의 완료한 상태이고, 그에따른 profile api 도 전달 받았습니다. (개인정보 정상적으로 전달 받음)

문의드린 부분은 그 이후에 api 전송하면 위와같은 에러 메세지를 응답하며 진행되지 않는 부분입니다.
(특정 계정에서만 현재 발생하는 것으로 파악됨 id : wc4r9q)

확인 한 번 부탁드립니다.
감사합니다.
